cla_public/templates/moj_template_base.jinja

```diff
@@ -31,153 +41,165 @@
-                {% else %}
-                  Built by <a class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr title="Ministry of Justice">MoJ</abbr> Digital Services</a>
-                {% endif %}
-              </li>
-            </ul>
-
-            <svg role="presentation" focusable="false" class="govuk-footer__licence-logo" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 483.2 195.7" height="17" width="41">
-              <path fill="currentColor" d="M421.5 142.8V.1l-50.7 32.3v161.1h112.4v-50.7zm-122.3-9.6A47.12 47.12 0 0 1 221 97.8c0-26 21.1-47.1 47.1-47.1 16.7 0 31.4 8.7 39.7 21.8l42.7-27.2A97.63 97.63 0 0 0 268.1 0c-36.5 0-68.3 20.1-85.1 49.7A98 98 0 0 0 97.8 0C43.9 0 0 43.9 0 97.8s43.9 97.8 97.8 97.8c36.5 0 68.3-20.1 85.1-49.7a97.76 97.76 0 0 0 149.6 25.4l19.4 22.2h3v-87.8h-80l24.3 27.5zM97.8 145c-26 0-47.1-21.1-47.1-47.1s21.1-47.1 47.1-47.1 47.2 21 47.2 47S123.8 145 97.8 145" />
-            </svg>
-            <span class="govuk-footer__licence-description">
+    <div class="govuk-header__content">
+      <a href="/" class="govuk-header__link govuk-header__link--service-name">
+        {{ proposition_title }}
+      </a>
+    </div>
+  </div>
+</header>
+<!--end header-->
+<div class="govuk-width-container">
+  <!--[if lte IE 6]>
+  <script>document.body.className = document.body.className.replace('js-enabled', '');</script><![endif]-->
+  {% block before_content %}{% endblock %}
+  {% block content %}{% endblock %}
+</div>
+
+<footer class="govuk-footer " role="contentinfo">
+  <div class="govuk-width-container ">
+    {% block footer_top %}{% endblock %}
+    <div class="govuk-footer__meta">
+      <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+        <h2 class="govuk-visually-hidden">Support links</h2>
+        <ul class="govuk-footer__inline-list">
+          <li class="govuk-footer__inline-list-item">
+            {% if request.cookies.get('locale') == 'cy_GB' %}
+              <a class="govuk-footer__link" lang="en" href="{{ url_for('base.set_locale', locale='en_GB') }}"
+                 data-ga="event:locale/switch/english">English</a>
+            {% else %}
+              <a class="govuk-footer__link" lang="cy" href="{{ url_for('base.set_locale', locale='cy_GB') }}"
+                 data-ga="event:locale/switch/welsh">Cymraeg</a>
+            {% endif %}
+          </li>
+          {% block footer_support_links %}{% endblock %}
+          <li class="govuk-footer__inline-list-item">
+            {% if request.cookies.get('locale') == 'cy_GB' %}
+              Hadeiladwyd gan<a class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr
+              title="Gweinyddiaeth Cyfiawnder">MoJ</abbr> Gwasanaethau Digidol</a>
+            {% else %}
+              Built by<a class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr
```

Comment:
This line removes a space from the text, causing the footer of the site to read 
> Built byMoJ Digital Services

The line was changed from the previous line (Line number 149- which is red above)
`Built by <a class="govuk-footer__link" href="https://mojdigital.blog.gov.uk/"><abbr title="Ministry of Justice">MoJ</abbr> Digital Services</a>`